benchmarks/dynamo/common.py

```diff
@@ -1364,10 +1375,14 @@ def load(cls, model, example_inputs, device):
             else:
                 _register_dataclass_output_as_pytree(example_outputs)
 
+            combined_args = tuple(example_args) + tuple(example_kwargs.values())
```

Comment:
curious, no need to look at the model signature here?

Reply:
That could be better if the kwargs are out of order, but I don't think that's happening here. We'll just feed the shapes spec as a tuple.